frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -106,6 +113,15 @@ public final class EnhancedGraphTable extends BaseEnhancedGraphTable
   @UIEffect
   private @Nullable String selectedBranchName;
 
+  // To accurately track the change of the current branch from the begging, let's put something impossible
+  // as a branch name. This is required to detect the moment when the unmanaged branch notification should be shown.
+  @UIEffect
+  private String mostRecentlyCheckedOutBranch = "?!@#$%^&";
+  @UIEffect
+  private @Nullable Notification slideInNotification;
+  @UIEffect
```

Comment:
Pls remove this annotation, unfortuntately Checker doesn't seem to enforce that `@UIEffect` fields can only be accessed/assigned from `@UIEffect` methods as I thought ☹️

Reply:
remove **this** only annotation, remove `@UIEffect` annotations introduced by myself or remove all `@UIEffect` on fields within `EnhancedGraphTable.java`?

Reply:
I'll push the fix myself!